blue-team/linux/update_passwords.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# change password for users 
+grep "/home" /etc/passwd | awk -F: '{print $1}' > users.txt
+while read user; do
+	echo "$user@123" | passwd --stdin "$user"
```

Comment:
we should read in the appended password, as if we edit the file, we might accidently leave it if we forget to delete the script

Another note (thanks @advilm ) we should use `${user}` rather than just `$user` as this is better practice